vm/runtime.ts

```diff
@@ -421,7 +421,7 @@ module J2ME {
     ctx: Context;
     allCtxs: Set<Context>;
 
-    isolate: com.sun.cldc.isolate.Isolate;
+    isolate: any;
```

Comment:
> Why is this any now?

I reverted this change in e854d96, since the code no longer references runtime.isolate._priority, but I did something similar in aa6cbc3, so you'll have the same question about that.

The reason is that the *com.sun.cldc.isolate.Isolate* type doesn't appear to be defined anywhere. Apparently, that's fine with TypeScript, as long as you don't attempt to access any of its properties. But once you do, the TypeScript compiler will complain the property:

> tsc --sourcemap --target ES5 references.ts -d --out bld/j2me.js
> vm/jvm.ts(55,34): error TS2339: Property '_priority' does not exist on type 'Isolate'.
> make: *** [bld/j2me.js] Error 3

I suppose there's a right thing to do here, like declaring that type. I just lack the TypeScript knowledge to immediately know what it is (and where to put it, etc.). And I figured that since we're using the type without actually defining it anywhere, it doesn't provide much value. But I may well be missing something.